core/CMakeLists.txt

```diff
@@ -129,17 +131,51 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${OUTPUT})
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${OUTPUT})
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${OUTPUT})
 
+#asmjit
+
+if (NOT DEFINED ASMJIT_EMBED)
+    set(ASMJIT_EMBED TRUE)
+endif()
+
+include(FetchContent)
+FetchContent_Declare(
+        asmjit
+        GIT_REPOSITORY https://github.com/questdb/asmjit.git
```

Comment:
Does this always take latest? Shall we use release tag?

Reply:
It does take latest, but it's our own fork "under control".

Reply:
It's still better to use tag on our fork. Say if you update it, you don't want old releases compilations to start failing